fedramp-integrated-inventory-workbook/LICENSE

```diff
@@ -0,0 +1,15 @@
+Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
We normally use Apache 2, think this is compatible for re-license but not a lawyer